### DISCUSSION TITLE
Using RavenDB with Blazor Server and dependency injection

### DISCUSSION THREAD

**Discussion by divvan** [ravendb/ravendb#16310] [Q&A] [answered]
I'm an absolute beginner with Asp.Net and I have a very basic question on how to create a Blazor project with RavenDB using dependency injection. I've looked at Ayende's post and the RavenDB.DependencyInjection project:

https://ayende.com/blog/187906-B/using-ravendb-unit-of-work-and-net-core-mvc
https://github.com/JudahGabriel/RavenDB.DependencyInjection

They both mention a ConfigureServices method in Startup.cs, but the default Blazor Sever project seems to use Program.cs instead. So, I'm wondering how or where ConfigureServices should be implemented?

**Comment by garayx** [comment, marked as answer]
Hi,

There is an article that demonstrate how to build app using Blazor and RavenDB:

https://ravendb.net/articles/building-a-call-for-papers-website-with-blazor-ravendb
